client/src/containers/CompareContainer.js

```diff
@@ -27,6 +30,26 @@ function CompareContainer(props) {
     return catSettings.includes(o.espnId) && o.name !== 'mins'
   })
 
+  // Calculating Head to Head matchup table
```

Comment:
If you can refactor it to loop over the teams to reduce the copy paste code for each team. It might be tricky doing the calculate matchup though so depends on the ease of that.

 In the future adding 3+ teams for comparisons could be an idea too.

Reply:
Looking into this

Reply:
Something similar is done with the wins/min/max/mean summary table. One loop to init the values, then a second one to do the comparison.

`for (const team of selectedTeams) {` this block